android/app/src/main/java/org/uwaterloo/subletr/pages/createlisting/CreateListingPageViewModel.kt

```diff
@@ -120,16 +123,15 @@ class CreateListingPageViewModel @Inject constructor(
 							description = it.description,
 							residenceType = ResidenceType.house,
 							imgIds = imgIds,
-							roomsAvailable = it.numBedrooms,
-// TODO: ADDING LOCATION
-							latitude = 0f,
-							longitude = 0f,
-//							TODO: Update UI to allow the inputs of these newly added values
+							latitude = location!!.latitude.toFloat(),
```

Comment:
Plz, let's not crash the app. Plz.

Reply:
I think something went wrong during rebasing? The latitude/longtitude = 0 is the correct value on master.